webapp/src/main/java/com/box/l10n/mojito/common/notification/SlackMessageBuilder.java

```diff
@@ -0,0 +1,50 @@
+package com.box.l10n.mojito.common.notification;
+
+import static com.box.l10n.mojito.slack.SlackClient.COLOR_WARNING;
+
+import com.box.l10n.mojito.entity.TMTextUnit;
+import com.box.l10n.mojito.slack.request.Attachment;
+import com.box.l10n.mojito.slack.request.Field;
+import com.box.l10n.mojito.slack.request.Message;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+/** Build custom Slack warning messages below. */
+@Component
```

Comment:
Should probably make the presence of this component configurable, see `@ConditionalOnProperty`